terraform/waf.tf

```diff
@@ -0,0 +1,122 @@
+﻿resource "aws_wafv2_regex_pattern_set" "ehrc_protected_request_address" {
+  name        = "ehrc-rotected-request-address"
+  description = "Regex of the endpoint used by ehrc. Of the form .../download?p=filename"
+  scope       = "REGIONAL"
+
+  regular_expression {
+    regex_string = "^/download(/)?[?]p=(.*)$"
```

Comment:
The `/` might need to be escaped in this. I would also normally use `\` to escape a special character (i.e. `\?` instead of `[?]`)

Reply:
I've tried to do it, and played around with escaping characters but terraform won't accept escaping neither / nor ?, so I don't know if I can do it in another way.

Reply:
Weird. As long as this works I'm happy with it then, but I'm surprised you can't escape them with a backslash.